MonoGame.Framework/MathHelper.cs

```diff
@@ -266,13 +266,13 @@ public static float SmoothStep(float value1, float value2, float amount)
         /// <param name="radians">The angle in radians.</param>
         /// <returns>The angle in degrees.</returns>
         /// <remarks>
-        /// This method uses double precission internally,
+        /// This method uses double precision internally,
         /// though it returns single float
         /// Factor = 180 / pi
         /// </remarks>
         public static float ToDegrees(float radians)
         { 
-            return (float)(radians * 57.295779513082320876798154814105);
```

Comment:
`57.295779513082320876798154814105` is not the same as `57.295779513082320876798154814105f`, the later is less precise. Return the original code.

Reply:
For reference on what actually happens to constant numbers: https://dotnetfiddle.net/aBz4fx